apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/login-button/edit.js

```diff
@@ -49,10 +49,36 @@ function LoginButtonEdit( props ) {
 
 	const colorProps = getColorAndStyleProps( attributes );
 
+	if ( Block ) {
+		return (
+			<>
+				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
+				<Block.div className="wp-block-button">
+					<RichText
+						placeholder={ __( 'Add text…', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **103** times:
`translate( 'Add text…' )` *ES Score: 9*